doc/book/src/patterns/id-pointer.md

```diff
@@ -0,0 +1,19 @@
+# ID Pointer
+
+ID Pointer is a technique which allows separation of the main data (an object) and its accessors / capabilities by linking the latter to the original. There's a few different directions in which this pattern can be used:
```

Comment:
```suggestion
ID Pointer is a technique that separates the main data (an object) and its accessors / capabilities by linking the latter to the original. There's a few different directions in which this pattern can be used:
```

Is there a different word for accessors? And it is odd to say that it separates the data by linking the things together. Is isolate correct? Is it that ID Pointer creates a link between objects and accessors?